src/main/kotlin/org/rust/lang/core/psi/ext/RsLooplikeExpr.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import org.rust.lang.core.psi.RsExpr
+
+/**
+ * An expression in which [RsLoopOnlyExpr] are valid. Currently the following expressions are [RsLooplikeExpr]: `loop`,
+ * `for`, and `while`.
```

Comment:
It's not true because `break` expr can be used in blocks with `label_break_value` feature

Reply:
Yeah, I meant to phrase that differently :/
I have now reworded and expanded the documentation comment.